helix-term/src/keymap.rs

```diff
@@ -271,6 +280,10 @@ impl Keymap {
                 self.state.clear();
                 KeymapResult::Matched(command)
             }
+            Some(&KeyTrie::Sequence(ref commands)) => {
```

Comment:
`Sequence` will also need to be handled at the beginning of this function:

```rust
        let trie = match self.root.search(&[first]) {
            Some(&KeyTrie::Leaf(cmd)) => return KeymapResult::Matched(cmd),
	        Some(&KeyTrie::Sequence(seq)) => return KeymapResult::MatchedSequence(seq),		
            None => return KeymapResult::NotFound,
            Some(t) => t,
        };
```

Reply:
Done